README.md

```diff
@@ -391,23 +391,23 @@ this project in the
 [OSSF Best Practices Working Group](https://github.com/ossf/wg-best-practices-os-developers)
 meetings.
 
-See the
-[Community Calendar](https://calendar.google.com/calendar?cid=czYzdm9lZmhwNWk5cGZsdGI1cTY3bmdwZXNAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ)
-for the schedule and meeting invitations. The meetings [happen biweekly](https://calendar.google.com/calendar/embed?src=s63voefhp5i9pfltb5q67ngpes%40group.calendar.google.com&ctz=America%2FLos_Angeles).
 
-The biweekly meeting minutes document can be found [here](https://docs.google.com/document/d/1dB2U7_qZpNW96vtuoG7ShmgKXzIg6R5XT5Tc-0yz6kE/edit#) (join ossf-scorecard-dev@googlegroups.com for access).
 
-[Join](https://support.google.com/groups/answer/1067205?hl=en) our email groups:
+| Artifact                      | Link                       |
+| --------------------------    | -------------------------- |
+| Scorecard Dev Forum           | [ossf-scorecard-dev@](https://groups.google.com/g/ossf-scorecard-dev) |
+| Scorecard Announcements Forum | [ossf-scorecard-announce@](https://groups.google.com/g/ossf-scorecard-announce) |
+| Community Meeting VC          | [https://zoom.us/j/98835923979](https://zoom.us/j/98835923979) |
```

Comment:
is it standard to provide easy-to-crawl/parse meeting links this way? No fear of zoom-bombing or the like? I don't think we can do anything about it, but if you come across a solution, let's use it in the future